src/List/ListItem.js

```diff
@@ -41,6 +42,7 @@ function getStyles(props, context, state) {
       !state.rightIconButtonKeyboardFocused ? hoverColor : null,
       color: textColor,
       display: 'block',
+      cursor: disabled ? 'not-allowed' : 'inherit',
```

Comment:
Well, you raised a good point. Should we change the default style when `disabled` is set to `true`?

The `<ListItem />` is really handy when we just want to display a list but don't want our items to be interactive.
I believe it's a valid use case. It may be not.
For that use case, we need to set `disabled={true}` and changing the `cursor` would be wrong.